app/views/setup/discovery.html.slim

```diff
@@ -0,0 +1,34 @@
+.alert.alert-info.alert-dismissible role="alert"
+  button.close[type="button" data-dismiss="alert" aria-label="Close"]
+    span[aria-hidden="true"]
+      | &times;
+  i.fa.fa-4x.pull-left aria-hidden="true"
+  span
+    | After choosing the master and clicking "Bootstrap cluster" all other nodes will be set to the minion role
+h1 Nodes
+.row
+  .col-xs-12.discovery-control
+    span #{Minion.count} Nodes found
+= form_tag(setup_bootstrap_path, method: "post")
+.nodes-container
```

Comment:
Shouldn't `.nodes-container` be indented 1 extra level to be contained by the form?